### DISCUSSION TITLE
Update to 4.10.0-0.okd-2022-06-10-131327 stuck in Api server operator update

### DISCUSSION THREAD

**Discussion by leonardoparejau** [okd-project/okd#1262] [Q&A]
**Describe the bug**
Im trying to perform a global update from 4.10.0-0.okd-2022-05-28-062148 to 4.10.0-0.okd-2022-06-10-131327 .
The update is stuck for a long period of time. The cluster version operator pod shows the following message:

`E0617 09:51:24.396418       1 task.go:112] error running apply for clusteroperator "kube-apiserver" (99 of 778): Cluster operator kube-apiserver is updating versions
E0617 09:51:43.259718       1 task.go:112] error running apply for clusteroperator "kube-apiserver" (99 of 778): Cluster operator kube-apiserver is updating versions
I0617 09:51:46.571660       1 metrics.go:480] skipping metrics for etcd ClusterOperator condition RecentBackup=Unknown (neither True nor False)
E0617 09:52:28.508386       1 task.go:112] error running apply for clusteroperator "kube-apiserver" (99 of 778): Cluster operator kube-apiserver is updating versions
`

Also the Kube api server operator pod, shows the following: 
`I0617 09:53:31.900316       1 event.go:285] Event(v1.ObjectReference{Kind:"Deployment", Namespace:"openshift-kube-apiserver-operator", Name:"kube-apiserver-operator", UID:"c0c7993a-440c-4093-ac4e-708d18be6a08", APIVersion:"apps/v1", ResourceVersion:"", FieldPath:""}): type: 'Normal' reason: 'MultipleVersions' multiple versions found, probably in transition: quay.io/openshift/okd-content@sha256:30d843741c26dcea78e1fb3a653ade8db7825057e82920b76d960e5603812c15,quay.io/openshift/okd-content@sha256:8303284872046f2f89b26effe6daf880eebffe716342c397108c02bd04fb9ba9`

It stays stopped on this stage:
Working towards 4.10.0-0.okd-2022-06-10-131327: 98 of 778 done (12% complete)

must-gather-logs on the following link:
https://drive.google.com/file/d/1IacclSMDw8QFFB245sn-TAGNiLeDE-P8/view?usp=sharing

**Comment by vrutkovs** [comment]
```
2022-06-17T11:54:57.587529889+02:00 E0617 09:54:57.587429       1 degraded_webhook.go:128] x509: certificate signed by unknown authority
```
in kube-apiserver operator log is suspicious. Please file a new bug at https://bugzilla.redhat.com/enter_bug.cgi?component=kube-apiserver&product=OpenShift%20Container%20Platform&version=4.10
